tmt/steps/provision/testcloud.py

```diff
@@ -361,8 +368,9 @@ def _get_url(self, url, message):
             except requests.RequestException:
                 pass
             if timeout < 0:
-                raise ProvisionError(
-                    f'Failed to {message} in {DEFAULT_CONNECT_TIMEOUT}s.')
+                error_msg = f'Failed to {message} in {DEFAULT_CONNECT_TIMEOUT}s.'
```

Comment:
See the comment for __init__ - I think it should be printed there when correctly expecting provisionerror. Non need to change testcloud.py for the error messages.